modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -576,14 +576,16 @@ public class IgniteImpl implements Ignite {
 
         // TODO: IGNITE-19344 - use nodeId that is validated on join (and probably generated differently).
         txManager = new TxManagerImpl(
+                name,
```

Comment:
It could be used, but there is a nuance: `clusterSvc.topologyService().localMember().name()` can only be used during the start of components and not in the constructor, for now I will leave the constructor parameter.